src/hmrc-content-guide/recognised-overseas-pension-schemes-notification-list/index.njk

```diff
@@ -0,0 +1,39 @@
+---
+title: Recognised overseas pension schemes notification list
+menuText: Recognised overseas pension schemes notification list
+layout: twoColumnPage.njk
+---
+{% from "_example.njk" import example %}
+
+<h2 class="govuk-heading-l">Document type</h2>
+
+<p class="govuk-body">Detailed guide</p>
+
+<h2 class="govuk-heading-l">When to use</h2>
+
+<p class="govuk-body">
+Use this pattern when updating <a href="https://www.gov.uk/guidance/check-the-recognised-overseas-pension-schemes-notification-list">Check the recognised overseas pension schemes notification list</a>.
+</p>
+
+<h2 class="govuk-heading-l">How it works</h2>
+
+<ol class="govuk-list govuk-list--number">
+<li>Delete all the 'Changes' tables in the list</li>
+<li>Add, amend or delete the relevant entries on the ROPS list</li>
+<li>Add a 'Changes' table for each country you update, and for the change use 'Added', 'Amended' or 'Deleted' - if the name of a scheme has changed, put the old name in brackets, for example: 'Booster SuperScheme (previously Fidelity Super-Super Plan)'</li>
+<li>Do not include any company names in the change note, for example 'The recognised overseas pension schemes notification list has been updated, with (number) schemes added and (number) removed'</li>
+</ol>
+
+<h3 class="govuk-heading-m">Where to place this component</h3>
```

Comment:
Is this an H2 or is it still part of 'How it works'?

Reply:
I've tried to leave the content design to the GOV.UK designers, but have just changed this one now as it needs optimising.

I've just removed the H3 and its text, instead slightly amending one of the list items to include where the change table should be placed.